code-challenges.rb

```diff
@@ -9,6 +9,22 @@
 # Expected output: '42 is even'
 num3 = 221
 # Expected output: '221 is odd'
+num4 = 0
+# Expected output: 'num is 0'
+
+def only_odd num
+  if num == 0
+    'num is 0'
+  elsif num.even?
+    "#{num} is even"
+  else num.odd?
+    "#{num} is odd"
+  end
+end
+p only_odd num1
+p only_odd num2
+p only_odd num3
+p only_odd num4
```

Comment:
I like the approach here. The only issue I am seeing is passing a condition on your else statement on line 20. That should prolly be an `elsif`